CONTRIBUTING.md

```diff
@@ -3,37 +3,140 @@
 This is intended for developers only, see [README](README.md) for the
 main docs.
 
-## Installation
+## Development principles
 
-For running locally without Docker you will need
+### Project’s aims
 
- * robot
- * owltools
- * python3.6 or higher
+The primary aim of the ODK (its mission statement, if you will) is to
+make it possible for any ontology project to benefit from thoroughly
+designed, well tested ontology engineering workflows, even if the
+project does not have the luxury of a full-time ontology pipeline
+engineer always available to fix snafus and keep things running.
 
-See [Dockerfile](Dockerfile) for details on how to obtain these
+From this aim, we derive the following guidelines that developers should
+keep in mind whenever working on the ODK (especially when adding new
+features):
 
-## How it works
+(1) No feature should require from the user to install anything on their
```

Comment:
technically (just being a bit extreme now) you will need `bash` or similar as well :P